Makefile

```diff
@@ -186,8 +186,8 @@ kindleupdate: all
 	# note that the targz file extension is intended to keep ISP from caching
 	# the file, see koreader#1644.
 	cd $(INSTALL_DIR) && \
-		tar czafh ../koreader-$(DIST)-$(MACHINE)-$(VERSION).targz \
-		-T koreader/ota/package.index --no-recursion
+		tar --no-recursion -czahf ../koreader-$(DIST)-$(MACHINE)-$(VERSION).targz \
```

Comment:
Might be clearer to write `-czahf --no-recursion` instead?

Reply:
I have a tendency to like having the tarball filename directly following the 'f' arg, but maybe that's just me?

Reply:
Noope, that's not just me, that's tar itself ^^.

Reply:
That's with tar 1.30, FWIW.

Reply:
Sure, good point. But, and this is just my opinion, I think in that case it's clearer to write something like `-czah --no-recursion -f <filename>`.

Reply:
Appears to work, and I have no objection/preference, so, let's :).